build.gradle

```diff
@@ -3,13 +3,14 @@ plugins {
     id "checkstyle"
     id "maven-publish"
     id "org.embulk.embulk-plugins" version "0.4.2"
+    id "com.palantir.git-version" version "0.12.3"
 }
 repositories {
     mavenCentral()
 }
 
 group = "io.trocco"
-version = "0.3.1"
+version = versionDetails().lastTag
```

Comment:
Rubygems does not allow pushing gems that have the same version as the past version, so gems cannot be released without tagging new one.